configs/metadata.yaml

```
@@ -184,3 +184,38 @@ sentinel-1-rtc:
     wavelength:
       vv: 3.5
       vh: 4.0
+modis:
+  band_order:
+    - sur_refl_b01
+    - sur_refl_b02
+    - sur_refl_b03
+    - sur_refl_b04
+    - sur_refl_b05
+    - sur_refl_b06
+    - sur_refl_b07
+  gsd: 500
```

We would also need an rgb index.
```suggestion
     rgb_indices:
      - 0
      - 3
      - 2
```